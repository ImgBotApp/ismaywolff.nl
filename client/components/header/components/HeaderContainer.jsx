import glamorous from 'glamorous'

const HeaderContainer = glamorous.header({
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'baseline',
  paddingBottom: '1rem',
  flexWrap: 'wrap',

  '@media (min-width: 20em)': {
    flexWrap: 'unset'
  }
})

export default HeaderContainer
