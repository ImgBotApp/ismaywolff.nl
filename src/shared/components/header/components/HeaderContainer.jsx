import styled from 'styled-components'

const HeaderContainer = styled.header`
  margin-bottom: 2rem;

  @media (min-width: 30em) {
    align-items: center;
    display: flex;
    justify-content: space-between;
  }
`

export default HeaderContainer
