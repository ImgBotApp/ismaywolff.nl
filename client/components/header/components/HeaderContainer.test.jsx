import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import { matcher, serializer } from 'jest-glamor-react'
import HeaderContainer from './HeaderContainer'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

describe('<HeaderContainer />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HeaderContainer />)
    expect(shallowToJson(wrapper)).toMatchSnapshotWithGlamor()
  })
})
