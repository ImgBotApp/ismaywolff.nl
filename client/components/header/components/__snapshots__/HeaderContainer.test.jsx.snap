// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderContainer /> renders correctly 1`] = `
.css-1xnt1rf,
[data-css-1xnt1rf] {
  -webkit-flex-wrap: wrap;
  -webkit-box-lines: multiple;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -webkit-align-items: baseline;
  -ms-flex-pack: center;
  -ms-flex-wrap: wrap;
  justify-content: center;
  flex-wrap: wrap;
  padding-bottom: 1rem;
  align-items: baseline;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
}

<header
  className="css-1xnt1rf"
/>
`;
